packages/typography/typography.tsx

```diff
@@ -0,0 +1,44 @@
+import * as classnames from 'classnames';
+import * as React from 'react';
+
+export interface TypographyProps<T> extends React.HTMLProps<T> {
+  children?: React.ReactNode,
```

Comment:
Mostly stylistic but I'm fairly certain you can/should use semicolons instead of commas for terminating interface types.

https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties